src/ripple/plugin/exports.h

```diff
@@ -88,6 +95,14 @@ struct TransactorExport
     checkPriorTxAndLastLedgerPtr checkPriorTxAndLastLedger = nullptr;
     checkFeePtr checkFee = nullptr;
     checkSignPtr checkSign = nullptr;
```

Comment:
Can you also update the formatting in the rest of the struct?

Reply:
done